engines/wintermute/base/gfx/opengl/meshx.cpp

```diff
@@ -401,11 +428,16 @@ bool MeshX::parseNormalCoords(XFileLexer &lexer) {
 	uint vertexNormalCount = readInt(lexer);
 	assert(vertexNormalCount == _vertexCount);
 
+	_vertexNormalData = new float[3 * _vertexCount]();
```

Comment:
It should be guaranteed, the load function of ModelX is only called once in two different places after a new instance has been created. But these arrays could still be replaced by Common::Array, although I would prefer to do this at a later point, since not all of them were introduced in this pr.

Reply:
I'm fine with this, but please note a TODO to handle it.